.evergreen/generate_evergreen_tasks.js

```diff
@@ -599,8 +599,22 @@ BUILD_VARIANTS.push({
 
 const oneOffFuncAsTasks = []
 
+let testingFLEBranch = 'NODE-4433-createEncryptedCollection'
+
+function getPinnedCommit() {
+  if (testingFLEBranch == null) {
+    return '67bec571c0c21f4db8a96b6bd61cb24dfc87a223'
+  }
+  const {execSync} = require('child_process');
+  const latestCommitSha = execSync(`gh api "/repos/mongodb/libmongocrypt/commits?sha=${testingFLEBranch}" --jq '.[0].sha'`, {encoding: 'utf8'}).trim();
```

Comment:
I'm not in favor of requiring developers to have the `gh` package installed (I do not), especially since we can't add it as a "dev dependency" and install it on engineers' machines automatically.  Can we either revert this or figure out another solution that uses standard builtins?

Reply:
Sorry this isn't meant to merge, it just makes updating the commit quicker